components/SideBar/SideBarSavedPosts.tsx

```diff
@@ -6,14 +6,14 @@ import SideBarSavedArticlePreview from "./SideBarSavedArticlePreview";
 import Link from "next/link";
 
 export default React.memo(function SideBarSavedPosts() {
-  let { data: bookmarks } = api.post.myBookmarks.useQuery();
-  const { status: bookmarkStatus } = api.post.myBookmarks.useQuery();
+  const { data: bookmarksData } = api.post.myBookmarks.useQuery({
+    limit: 3,
+  });
+  const { status: bookmarkStatus } = api.post.myBookmarks.useQuery({});
```

Comment:
Can you please combine this query with the other one. This means we are making 2 queries when only one is necessary.